__tests__/app.test.js

```diff
@@ -3,6 +3,7 @@ const testData = require("../db/data/test-data");
 const db = require("../db/connection");
 const request = require("supertest");
 const app = require("../app");
+const { patch } = require("../app");
```

Comment:
What `patch` are you requiring here?

Reply:
hmm.. strange, I didn't actually write that, i'll delete it